src/coverlet.core/Helpers/RetryHelper.cs

```diff
@@ -1,3 +1,4 @@
+using Coverlet.Core.Abstracts;
```

Comment:
Here we simply make RetryHelper an "instance" type that implements IRetryHelper.